docs/feature_advanced_keycodes.md

```diff
@@ -2,7 +2,7 @@
 
 Your keymap can include keycodes that are more advanced than normal, for example shifted keys. This page documents the functions that are available to you.
 
-### Assigning Custom Names
+## Assigning Custom Names
```

Comment:
These level 3 headers were deliberate so they wouldn't make the TOC too confusing. With level 2 headers the TOC became big and hard to follow.

Reply:
I figured it was something like that.... 
And H3 ... get no links (nor love, apparently)